fireant/tests/slicer/widgets/test_csv.py

```diff
@@ -33,7 +33,7 @@ def test_single_metric(self):
         expected = single_metric_df.copy()[[fm('votes')]]
```

Comment:
Are there any tests specifically for the react table transformer?

Reply:
of course, in `fireant/tests/slicer/widgets/test_reacttable.py`

Reply:
Ah, github didn't load the diff as the file was too large.